doc/api/fs.md

```diff
@@ -2392,6 +2392,32 @@ changes:
 
 Synchronous lchown(2). Returns `undefined`.
 
+## `fs.lutimes(path, atime, mtime, callback)`
+<!-- YAML
+addded: REPLACEME
+-->
+
+* `path` {string|Buffer|URL}
+* `atime` {number|string|Date}
+* `mtime` {number|string|Date}
+* `callback` {Function}
+  * `err` {Error}
+
+Asynchronous lutimes(2). No arguments other than a possible exception
+are given to the completion callback.
+
+## `fs.lutimesSync(path, atime, mtime)`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `path` {string|Buffer|URL}
+* `atime` {number|string|Date}
+* `mtime` {number|string|Date}
+* `mode` {integer}
+
+Synchronous lutimes(2). Returns `undefined`.
```

Comment:
Mention that if setting the access and modification time of the file fails an Error is thrown?

Reply:
This should also link to the asynchronous version similarly to other methods, for example fs.utimesSync does:

> For detailed information, see the documentation of the asynchronous version of this API: fs.utimes().